.husky/pre-commit

```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+REGEX="^(develop|main|staging|((feature|fix|docs|chore|refactor|test)\/.+))$"
```

Comment:
can we replace 'feature' with 'feat'? I think people are used to 'feat' already, wdyt @arturbeg?

Reply:
@ioanaroceanu2000 thanks for the suggestion! I think we can add it as you suggest! I will ask in the dev channel for suggestion

Reply:
@ssbarbee feel free to merge this PR once 'feature' is replaced by 'feat':) make sure to update the branch before merging

Reply:
yes yes that makes sense! I think I will hold until we settle with LP Opt push and merge it sometime this week.

Reply:
sounds good